Makefile

```diff
@@ -82,7 +82,7 @@ TEST_REQUIRES_ROOT_PACKAGES=$(filter \
 
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress
-MANPAGES=ctr.1 containerd.1 containerd-config.1 containerd-config.toml.5
+MANPAGES=containerd-config.1 containerd-config.toml.5
```

Comment:
if these two generated manpages aren't in the `$MANPAGES` list, then the `install-man` target will not install them to `$(DESTDIR)/man`.  Can probably solve with some other target that looks for content in `man/` but will need to be fixed.

Reply:
Re-added those manpages back to this `MANPAGES` variable.